src/js/utils/rxjsUtils.ts

```diff
@@ -0,0 +1,30 @@
+import { timer } from "rxjs/observable/timer";
+
+import "rxjs/add/operator/delayWhen";
+import "rxjs/add/operator/scan";
+import { Scheduler } from "rxjs/Scheduler";
+import { Observable } from "rxjs/Observable";
+
+export function linearBackoff(
+  delay: number,
+  maxRetries: number = -1,
+  maxDelay?: number,
+  scheduler?: Scheduler
```

Comment:
Do we want to have `IScheduler` here instead? 🤔

Reply:
It will be perfectly fine, though underlying implementation expects `IScheduler`. Having `Scheduler` is just more strict where we want it to be more permissive.

Reply:
Ok, changed the type 👍